commit e5d90505ac2bcc7434ce793346318b0392332b0d (refs/changes/43/3343/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-09T17:46:19+00:00 (1 year, 8 months ago)
    
    TRITON-110 current vmapi not compatible with ancient workflow
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/lib/workflows/nfs-volumes/index.js b/lib/workflows/nfs-volumes/index.js
index 1af5a30..f443c65 100644
--- a/lib/workflows/nfs-volumes/index.js
+++ b/lib/workflows/nfs-volumes/index.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -704,31 +704,17 @@ module.exports = {
             name: 'volapi.provision_nfs_volumes',
             timeout: 120,
             retry: 1,
-            body: provisionNfsVolumes,
-            modules: {
-                sdcClients: 'sdc-clients',
-                uuid: 'uuid',
-                vasync: 'vasync',
-                VError: 'verror'
-            }
+            body: provisionNfsVolumes
         }, {
             name: 'cnapi.wait_for_nfs_volumes_provisions',
             timeout: 120,
             retry: 1,
-            body: waitForNfsVolumeProvisions,
-            modules: {
-                sdcClients: 'sdc-clients',
-                vasync: 'vasync'
-            }
+            body: waitForNfsVolumeProvisions
         }, {
             name: 'cnapi.build_nfs_volumes_metadata',
             timeout: 10,
             retry: 1,
-            body: buildNfsVolumesMetadata,
-            modules: {
-                sdcClients: 'sdc-clients',
-                vasync: 'vasync'
-            }
+            body: buildNfsVolumesMetadata
         }
     ],
     addVolumesReferences: addVolumesReferences,
